packages/patrol/example/integration_test/notifications_test.dart

```diff
@@ -4,18 +4,19 @@ import 'common.dart';
 
 void main() {
   patrol(
-    'sends a notification, verifies that it is visible and taps on it by text',
+    'taps on notification',
     ($) async {
       await $.pumpWidgetAndSettle(ExampleApp());
 
       await $('Open notifications screen').tap();
-      await $(RegExp('someone liked')).tap();
 
       if (await $.native.isPermissionDialogVisible()) {
```

Comment:
In a perfect world - probably yes, but safeguards like this help a lot.

Patrol CLI uninstalls the app after executing every `*_test.dart` file, but sometimes I run these tests without Patrol CLI (so the app is not uninstalled after tests finish).

Also, in the future, we plan to enable Hot Reload for integration tests. This implies not killing the app, and safeguards like this will be necessary then.